home.html

```diff
@@ -12,6 +12,7 @@
 </head>
 <body>
     <header>
+        <!--Navbar Section-->
     <nav>
         <div class="nav-header">
             <div class="nav-logo">
```

Comment:
Try using a different color for the hover effect for the nav-items. It should contrast the background-color of contact-us button.

Do not use same class name for id. How do we tell the difference?